src/analyses/malloc_null.ml

```diff
@@ -16,6 +17,12 @@ struct
   module C = ValueDomain.AddrSetDomain
   module P = IdentityP (D)
 
+  let ignore_asm = ref true
+
+  (*TODO: why is this init not called??? *)
+  let init _ =
+    ignore_asm := get_bool "asm_is_nop"
```

Comment:
The `init` of the analysis is only called when an incremental analysis is to be performed and the analysis should be initialized with some data from the previous analysis run.

Reply:
I have to correct myself here: The `init` is actually called, even in a non-incremental run, but with `None` as a paramter. I got confused by some comment in the PR. Apologies for that.